[Bug]: litellm 1.59.9 `ModuleNotFoundError: No module named 'cgi'`

### What happened?

Error showing in the latest version of litellm 1.59.9 `ModuleNotFoundError: No module named 'cgi'`



### Relevant log output

```shell
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_llm.py:9: in <module>
    from assistant.utils.llm import MODELS, LLMClient
assistant/utils/__init__.py:19: in <module>
    from assistant.utils.llm import MODELS, TOKENIZER, Chunk, LLMClient
assistant/utils/llm.py:11: in <module>
    import litellm
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/litellm/__init__.py:1061: in <module>
    from .llms.bedrock.chat.converse_transformation import AmazonConverseConfig
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/litellm/llms/bedrock/chat/__init__.py:1: in <module>
    from .converse_handler import BedrockConverseLLM
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/litellm/llms/bedrock/chat/converse_handler.py:20: in <module>
    from .invoke_handler import AWSEventStreamDecoder, MockResponseIterator, make_call
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/litellm/llms/bedrock/chat/invoke_handler.py:32: in <module>
    from litellm.litellm_core_utils.prompt_templates.factory import (
/opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/litellm/litellm_core_utils/prompt_templates/factory.py:2156: in <module>
    from cgi import parse_header
E   ModuleNotFoundError: No module named 'cgi'
```

### Are you a ML Ops Team?

Yes

### What LiteLLM version are you on ?

v1.59.9

### Twitter / LinkedIn details

https://www.linkedin.com/in/glenn-jocher/

--- a/litellm/litellm_core_utils/prompt_templates/factory.py
+++ b/litellm/litellm_core_utils/prompt_templates/factory.py
@@ -2153,7 +2153,7 @@ def stringify_json_tool_call_content(messages: List) -> List:
 
 import base64
 import mimetypes
-from cgi import parse_header
+from email.message import Message
 
 import httpx
 
@@ -2174,8 +2174,9 @@ from litellm.types.llms.bedrock import ToolUseBlock as BedrockToolUseBlock
 
 
 def _parse_content_type(content_type: str) -> str:
-    main_type, _ = parse_header(content_type)
-    return main_type
+    m = Message()
+    m['content-type'] = content_type
+    return m.get_content_type()
 
 
 class BedrockImageProcessor:

